scss/application.scss

```diff
@@ -0,0 +1,52 @@
+@import url(http://fonts.googleapis.com/css?family=Roboto);
+
+// Colors
+$blue: #00ADEE;
+$green: #BED23C;
+$grey: #C4C4C4;
```

Comment:
This is great but move the variables into a separate file. For example in this case:

```
/scss
| -- /variables
     | -- _colors.scss
```

And there you will store all variables  and them you can import them :smile:

Reply:
Do the same for the sizes

Reply:
BTW, this file should import all the styles and then you import this file in the HTML.